pkg/inject/sidecar.go

```diff
@@ -158,14 +160,24 @@ func container(jaeger *v1.Jaeger, dep *appsv1.Deployment) corev1.Container {
 		Image: util.ImageName(jaeger.Spec.Agent.Image, "jaeger-agent-image"),
 		Name:  "jaeger-agent",
 		Args:  args,
-		Env: []corev1.EnvVar{{
-			Name: envVarPodName,
-			ValueFrom: &corev1.EnvVarSource{
-				FieldRef: &corev1.ObjectFieldSelector{
-					FieldPath: "metadata.name",
+		Env: []corev1.EnvVar{
+			{
+				Name: envVarPodName,
+				ValueFrom: &corev1.EnvVarSource{
+					FieldRef: &corev1.ObjectFieldSelector{
+						FieldPath: "metadata.name",
+					},
 				},
 			},
-		}},
+			{
+				Name: envVarPodIp,
+				ValueFrom: &corev1.EnvVarSource{
+					FieldRef: &corev1.ObjectFieldSelector{
+						FieldPath: "status.podIP",
```

Comment:
I think the original request was to have the _host_ IP, not the pod. IIRC, the pod IP will already be reported as part of the process tags.

cc @TBBle

Reply:
Haaa you are totally right! Changed.